### PR TITLE
Add README and build script

### PR DESCRIPTION
**Pull request by pritishnahar95**

### CODE REVIEW

**Review by whoisj — APPROVED** [review]

File: components/inference-service/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+./build.sh
+```
+
+To build the MIS container manually, you can run:
+```bash
+docker build -f dockerfile -t monai/inference-service:0.1 .
+```
+
+### Helm Chart Configuration
+Helm charts are located in the charts folder.
```
Comment (gigony) [inline components/inference-service/README.md:36]: Usually, there is a blank line after section (`#`) and it is a valid syntax of Markdown though many parser can recognize this.

File: components/inference-service/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+In order to install the helm chart, please run:
+```bash
+helm install monai-inference-service ./charts
+```
+
+##  Submitting inference requests
+####  Making a request with `curl`
+
+With MIS running, a user can make an inference request to the service using the `/upload` POST endpoint with the **cluster IP** and **Port** from running `kubectl get svc` and a compressed .zip file containing all the input payload files (eg. input.zip)
```
Comment (gigony) [inline components/inference-service/README.md:86]: `eg.` -> `e.g.,`

File: components/inference-service/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+##### example:
+```bash
+curl -X 'POST' 'http://10.97.138.32:8000/upload/' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: multipart/form-data' \
+   -F 'file=@input.zip;type=application/x-zip-compressed' \
+   -o output.zip
+```
+
+To view the FastAPI generated UI for an instance of MIS, have the service running and then on any browser, navigate to `http://CLUSTER IP:PORT/docs` (eg. http://10.97.138.32:8000/docs)
```
Comment (gigony) [inline components/inference-service/README.md:106]: `eg.` -> `e.g.,`

File: components/inference-service/build.sh [hunk 4]
```diff
@@ -0,0 +1 @@
+docker build -f dockerfile -t monai/inference-service:0.1 .
```
Comment (gigony) [inline components/inference-service/build.sh:1]: Ends with a blank line

**Review by gigony — COMMENTED** [review]
Looks good to me. Left minor comments.

**Pull request merged**